chicago-regional/README.md

```diff
@@ -4,6 +4,8 @@ A large-scale, detailed representation of the Chicago region.
 ## Source
 Developed and provided by the Chicago Area Transportation Study (CATS)
 
+[Abbas Babazadeh](mailto:ababazadeh@ut.ac.ir) reports that two other credible versions of the Chicago Regional problem exist for which network data are identical to the first version given above, but demand matrices differ. The second version, known as FHWA 2009 project, has been provided by Michael Florian.The specific attributes of this version are as follows: total trips=1,429,901.18 ([click here](https://drive.google.com/file/d/1xVcL8G5C3xNeHMrsgIrgAPU5xWCnTaTn/view?usp=sharing) for demand data); optimul objective function value=17,016,655.2611360890; cost weights=0.0 (Toll and Distance). The Third version appears in "A Greedy Path-Based Algorithm for Traffic Assignment," Xie, J., Nie, Y., Liu, X., Transportation Research Record, Vol. 2672(48), pp. 33-44, 2018. For this version, total trips=1,429,895.88 ([click here](https://drive.google.com/file/d/13S3ThVdWCt8qCIYOknSHlGlqGJslWA6p/view?usp=sharing) for demand data), optimul objective function value=17,016,355.0629789010, and cost weights=0.0.
```

Comment:
I don't think it makes sense to have links to external sources here. It looks like something that will definitely break in the future.

Reply:
The issue of self citing is still there (minor issue), but the pull request should include the actual data. A link to somebody's google drive (for data that fits well in a git repo) is utterly inadequate

Reply:
Thank you. The email address has been removed.

Reply:
How can I resolve the next issue?

Reply:
The changes in the text need to go here: https://github.com/bstabler/TransportationNetworks/tree/master/chicago-regional, and not in the main readme file.

And add the two demand files to the Pull request (in the appropriate folder) as well.